packages/cloud_firestore/cloud_firestore_platform_interface/lib/src/settings.dart

```diff
@@ -16,6 +16,9 @@ class Settings {
     this.host,
     this.sslEnabled,
     this.cacheSizeBytes,
+    this.webExperimentalForceLongPolling = true,
+    this.webExperimentalAutoDetectLongPolling = true,
+    this.timeoutSeconds = 30,
```

Comment:
We should probably not set any default so we can rely on the default value of the underlying JS SDK.